client/tinkerbell/finder.go

```diff
@@ -0,0 +1,139 @@
+package tinkerbell
+
+import (
+	"context"
+	"encoding/json"
+	"net"
+
+	"github.com/pkg/errors"
+	"github.com/tinkerbell/boots/client"
+	tinkClient "github.com/tinkerbell/tink/client"
+	tpkg "github.com/tinkerbell/tink/pkg"
```

Comment:
🤦🏻

Reply:
copied from `packet/endpoints.go`, that wrapper should probably move to some subpackage of pkg

Reply:
I just thought it was comical :D